kotsu/registration.py

```diff
@@ -35,23 +32,23 @@ def _load(name: str):
     return fn
 
 
-class AgentSpec:
-    """A specification for a particular instance of an agent.
+class EntitySpec:
```

Comment:
could just call these `Spec` and `Registry` rather than prefixing with `Entity` since it doesn't really add information

Reply:
> could just call these `Spec` and `Registry` rather than prefixing with `Entity` since it doesn't really add information

Ye cool.